packages/util/tsconfig.build.json

```diff
@@ -10,7 +10,10 @@
       "allowJs": true,
       "noImplicitAny": false,
       "sourceMap": true,
-      "outDir": "built"
+      "outDir": "built",
+      "typeRoots": [
```

Comment:
Out of curiosity what does this fix - anything we should do differently?

Reply:
I was having consistent failures from `tsc` complaining about not finding types for lots of packages. This tells it where to look. Not sure why no one else has seen these errors.